.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,56 @@
+## Join the developer team
+* Help rid NewPipe of bugs by sending an e-mail saying you want to join to <a href="mailto:tnp@newpipe.schabi.org">tnp@newpipe.schabi.org</a> that says what you want to help with.
+You may be asked to register at our [Sentry](https://sentry.schabi.org) instance.
 
 ## Translation
 
-* NewPipe is translated via [Weblate](https://hosted.weblate.org/projects/newpipe/strings/). You can log in there
-with your GitHub account.
-* If the language you want to translate is not on Weblate, you can add it: see [How to add a new language](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-add-a-new-language-to-NewPipe) in the wiki.
+* NewPipe is translated via [Weblate](https://hosted.weblate.org/projects/newpipe/strings/).
+Log in there with your GitHub account, or register.
+* Add the language you want to translate if it is not there already: see [How to add a new language](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-add-a-new-language-to-NewPipe) in the wiki.
 
 ## Code contribution
 
-* If you want to add a feature or change one, please open an issue describing your change. This gives the team and community a chance to give feedback before you spend any time on something that could be done differently or not done at all. It also prevents two contributors from working on the same thing and one being disappointed when only one user's code can be added.
-* Stick to NewPipe's style conventions: follow [checkStyle](https://github.com/checkstyle/checkstyle). It will run each time you build the project.
-* Do not bring non-free software (e.g. binary blobs) into the project. Also, make sure you do not introduce Google
+* Add a feature or change one by opening an issue describing your change. This gives the team and community a chance to give feedback before you spend time on something already underway,could be done differently, or should be avoided.
+* Stick to NewPipe's style conventions of [checkStyle](https://github.com/checkstyle/checkstyle). It runs each time you build the project.
+* Do not bring non-free software (e.g. binary blobs) into the project. Make sure you do not introduce Google
   libraries.
 * Stick to [F-Droid contribution guidelines](https://f-droid.org/wiki/page/Inclusion_Policy).
-* Make changes on a separate branch with a meaningful name, not on the master neither dev branch. This is commonly known as *feature branch workflow*. You
-  may then send your changes as a pull request (PR) on GitHub.
-* When submitting changes, you confirm that your code is licensed under the terms of the
-  [GNU General Public License v3](https://www.gnu.org/licenses/gpl-3.0.html).
-* Please test (compile and run) your code before you submit changes! Ideally, provide test feedback in the PR
+* Make changes on a separate branch with a meaningful name, not on the _master_ nor the _dev_ branch. This is commonly known as *feature branch workflow*. You may then send your changes as a pull request (PR) on GitHub.
```

Comment:
```suggestion
* Make changes on a separate branch with a meaningful name, not on the _master_ or _dev_ branches. This is commonly known as *feature branch workflow*. You may then send your changes as a pull request (PR) on GitHub.
```

Reply:
Master and dev are just onto themselves singular branches. Will accept rationale for "or" over "nor".

Reply:
It can be either:
"not on the master branch or the dev branch"
or
"not on the master or dev branches"

Because 'branch' by itself can only refer to one branch, not multiple.

If you want to use nor, you would have to frame it differently: "neither the master branch nor the dev branch". When you negate an entire list with 'not', it carries over all the individual elements. So you would say:
"I like neither 1, nor 2, nor 3."
or
"I do not like 1, 2, or 3"

Reply:
```suggestion
* Make changes on a separate branch with a meaningful name, not on the _master_ branch or the _dev_ branch. This is commonly known as *feature branch workflow*. You may then send your changes as a pull request (PR) on GitHub.
```